joomla/40-backend-template
# Issue #350 fix: Change sidebar icon on toggle & load

Pull Request for Issue #350  .

### Summary of Changes
Icon should change on menu toggle link click. 

**Before fix open**
![before-fix-open](https://user-images.githubusercontent.com/30073416/40267539-cb99668c-5b7b-11e8-93eb-dfa49bad5c45.png)

**After fix open**
![after-fix-open](https://user-images.githubusercontent.com/30073416/40267542-d387c140-5b7b-11e8-9fb6-2509b95ae7cc.png)

**Before fix close**
![before-fix-close](https://user-images.githubusercontent.com/30073416/40267546-daa67e9e-5b7b-11e8-9742-b85821a62bf1.png)

**After fix close**
![after-fix-close](https://user-images.githubusercontent.com/30073416/40267547-e009f26c-5b7b-11e8-9fc8-7c5bcf2d2369.png)

### Testing Instructions

1. Log in to Admin area 
2. Click on **Toggle Menu** Link, icon should change after click. 
3. Refresh page and check the menu toggle status should not change and icon should be appropriate. 

### Expected result

For Open menu, icon should be -
![after-fix-open](https://user-images.githubusercontent.com/30073416/40267589-c6e02ada-5b7c-11e8-99a3-61acb74b65ff.png)

For Close Menu, icon should be - 
![after-fix-close](https://user-images.githubusercontent.com/30073416/40267592-d6b0fa34-5b7c-11e8-8bdb-a867719c0e03.png)

### Actual result

For Open menu, icon is -
![after-fix-open](https://user-images.githubusercontent.com/30073416/40267589-c6e02ada-5b7c-11e8-99a3-61acb74b65ff.png)

For Close Menu, icon is - 
![after-fix-close](https://user-images.githubusercontent.com/30073416/40267592-d6b0fa34-5b7c-11e8-8bdb-a867719c0e03.png)


### Documentation Changes Required - **No**
